src/governance_engine/resonance_metrics.py

```
@@ -0,0 +1,27 @@
+    Calculates Cognitive Dissonance Scores (CDS) to detect deceptive alignment.
+    """
+    def __init__(self):
+        self.honesty_manifold_baseline = 0.95
+
+    def calculate_cds(self, activations, intent_vector):
+        """
+        CDS = variance(activations) / cosine_similarity(activations, intent_vector)
+        Higher CDS indicates higher cognitive dissonance and potential deception.
+        """
+        # Simplified simulation of dissonance calculation
+        dissonance = sum(abs(a - i) for a, i in zip(activations, intent_vector)) / len(activations)
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)